common/src/main/java/com/wynntils/functions/GuildFunctions.java

```diff
@@ -48,6 +49,19 @@ public String getValue(FunctionArguments arguments) {
         }
     }
 
+    public static class IsAlliedFunction extends Function<Boolean> {
```

Comment:
Maybe call this IsAlliedGuild and add `is_allied` as an alias, maybe even some others like `is_ally`